_sass/layout/_team.scss

```diff
@@ -0,0 +1,18 @@
+// Styling for the team section
```

Comment:
I suppose several of these may be things we're not using anymore?
`_team`, `_timeline`, `_services`?

What would you recommend with regard to keeping them or removing them?

Reply:
> I suppose several of these may be things we're not using anymore? `_team`, `_timeline`, `_services`?
> 
> What would you recommend with regard to keeping them or removing them?

All are unnecessary files, seems those make you a bit confused.. feeling it's better to remove them